batch7crud-roll2/src/main/java/com/lftechnology/batch7crud/util/DBConnection.java

```diff
@@ -0,0 +1,34 @@
+package com.lftechnology.batch7crud.util;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+import java.sql.*;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Created by Romit Amgai <romitamgai@lftechnology.com> on 1/19/16.
+ */
+
+public class DBConnection {
+  private static final Logger logger = Logger.getLogger("dbConnectionLogger");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1312)